### PR TITLE
[4221706127] Add user avatars selection

### PR DESCRIPTION
**Pull request by rYuuk**
## Description

-  Added user avatar screen which shows all created avatars by logged in user, they can be customized or selected

#### Added

- User avatar selection screen
- API for fetching all user avatars
- API for getting the pre-created avatar gameobject
- Background image for selection screens

#### Updated

- Fix for refresh session not getting stored
- Refactored profile manager 

## How to Test

- Login -> it should take you to avatar selection
- Try customize and select an avatar

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

### CODE REVIEW

**Review by HarrisonHough — APPROVED** [review]
All good, no blockers just some comments and suggestions. 

Great stuff!

File: Runtime/WebRequests/AvatarAPIRequests.cs [hunk 1]
```diff
@@ -19,6 +21,21 @@ public AvatarAPIRequests(CancellationToken ctx = default)
...
+            var response = await authorizedRequest.SendRequest(
+                new RequestData
+                {
+                    Url = $"{Endpoints.AVATAR_API_V1}{FETCH_AVATAR_PARAMETERS}{userId}",
+                    Method = Method.GET,
+                },
+                ctx: ctx
+            );
+            var json = JObject.Parse(response.Text);
+            var data = json["data"]!.ToString();
```
[the remark this answers is not in the document]
Comment (rYuuk) [reply Runtime/WebRequests/AvatarAPIRequests.cs:35]: Ya, I wish we could change this. Yuri says this is standard 🤷🏽‍♂️

[the remark this answers is not in the document]
Comment (HarrisonHough) [reply Runtime/WebRequests/AvatarAPIRequests.cs]: well it's a standard pain in the ass

**Review by HarrisonHough — APPROVED** [review]
Super duper!

File: Runtime/WebRequests/WebRequestDispatcher.cs [hunk 2]
```diff
@@ -70,14 +70,14 @@ public static class WebRequestDispatcher
 
             if (request.result != UnityWebRequest.Result.Success)
             {
-                Debug.Log(request.downloadHandler.text);
+                Debug.Log(request.downloadHandler.text + "\n" + url);
```
Comment (srcnalt) [inline Runtime/WebRequests/WebRequestDispatcher.cs:73]: Can remove the debug line I assume.

**Review by srcnalt — APPROVED** [review]
All good. Next iteration we can also cache the images unless user updates the defaults list probably.

**Pull request merged**